voidcrew/modules/cloning/code/objects_datums.dm

```diff
@@ -0,0 +1,63 @@
+//objects
+
+/obj/effect/countdown/clonepod
+	name = "cloning pod countdown"
+	color = "#18d100"
+	text_size = 1
+
+/obj/effect/countdown/clonepod/get_value()
+	var/obj/machinery/clonepod/attacked_clone_pod = attached_to
+	if(!istype(attacked_clone_pod))
+		return
+	if(attacked_clone_pod.occupant)
+		var/completion = round(attacked_clone_pod.get_completion())
+		return completion
+
+//datums
+
+/datum/brain_trauma
+	var/clonable = TRUE // will this transfer if the brain is cloned?
+
+/datum/brain_trauma/proc/on_clone()
+	if(clonable)
+		return new type
+
+/datum/brain_trauma/mild/phobia/on_clone()
+	if(clonable)
+		return new type(phobia_type)
+
+/datum/brain_trauma/special/psychotic_brawling/bath_salts
+	clonable = FALSE
+
+/datum/brain_trauma/special/beepsky
+	clonable = FALSE
+
+/datum/mood_event/cloned_corpse
+	description = "<span class='boldwarning'>I recently saw my own corpse...</span>\n"
+	mood_change = -6
+
+/datum/quirk/proc/clone_data() //return additional data that should be remembered by cloning
+/datum/quirk/proc/on_clone(data) //create the quirk from clone data
```

Comment:
This also isnt used, since it's on quirk, not trauma.

Reply:
I have just found out they are used